src/stories/tokenBalanceInput.stories.tsx

```diff
@@ -0,0 +1,55 @@
+import TokenBalanceInputComponent, {
+  TokenBalanceInputProps,
+} from "../components/TokenBalanceInput"
+import { Meta, Story } from "@storybook/react"
+import Nu from "../static/icons/NuLight"
+import Keep from "../static/icons/KeepLight"
+import { useState } from "react"
+
+const iconMap = {
+  Nu: Nu,
+  Keep: Keep,
+}
+
+const Template: Story<TokenBalanceInputProps> = ({ icon, max, label }) => {
+  const [amount, setAmount] = useState<string | number>("")
+  // @ts-ignore
+  const Icon = iconMap[icon]
```

Comment:
Continuing my thought from the previous comment - we can create a new interface in `utils.ts`:

```
import { createIcon } from "@chakra-ui/icons"

export interface ChakraIconMap {
  [key: string]: ReturnType<typeof createIcon>
}
```

and then we can use it here and get rid of `@ts-ignore`:
```suggestion
const iconMap: ChakraIconMap = {
  Nu: Nu,
  Keep: Keep,
}

const Template: Story<TokenBalanceInputProps> = ({ icon, max, label }) => {
  const [amount, setAmount] = useState<string | number>("")
  const Icon = iconMap[icon]
```